packages/msw-addon/src/mswDecorator.js

```diff
@@ -1,6 +1,12 @@
 let api
 
-const IS_BROWSER = typeof global.process === 'undefined'
+const IS_BROWSER = () => {
```

Comment:
The execution environment cannot change on runtime, so wrapping this logic in a function seems redundant.

Reply:
I just didn't like the idea of `IS_BROWSER` not being a constant, so I though a function would do it

Reply:
I see. It should be alright, as `IS_BROWSER` _is_ a constant and cannot change on runtime.